.github/workflows/cla.yml

```diff
@@ -1,12 +1,41 @@
 name: CLA Assistant
 
 on:
-  issue_comment:
-    types: [created]
-  pull_request_target:
-    types: [opened, synchronize]
+    issue_comment:
+        types: [created]
+    pull_request_target:
+        types: [opened, closed, synchronize]
 
 jobs:
-  CLA:
-    uses: Expensify/GitHub-Actions/.github/workflows/cla.yml@main
-    secrets: inherit
+    CLA:
+        runs-on: ubuntu-latest
+        # This job only runs for pull request comments or pull request target events (not issue comments)
+        if: github.event.issue.pull_request || github.event_name == 'pull_request_target'
+        steps:
+            - uses: actions-ecosystem/action-regex-match@9c35fe9ac1840239939c59e5db8839422eed8a73
+              id: sign
+              with:
+                  text: ${{ github.event.comment.body }}
+                  regex: '\s*I have read the CLA Document and I hereby sign the CLA\s*'
+
+            - uses: actions-ecosystem/action-regex-match@9c35fe9ac1840239939c59e5db8839422eed8a73
+              id: recheck
+              with:
+                  text: ${{ github.event.comment.body }}
+                  regex: '\s*recheck\s*'
+
+            - name: CLA Assistant
+              if: ${{ steps.recheck.outputs.match != '' || steps.sign.outputs.match != '' || github.event_name == 'pull_request_target' }}
```

Comment:
This condition was always evaluating to true because there were characters outside of the `${{ }}`